lib/rules/scrollable-region-focusable-matches.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Note:
+ * `excludeHidden=true` for this rule, thus considering only elements in the accessibility tree.
+ */
+const { isHidden, querySelectorAll } = axe.utils;
+const nodeAndDescendents = querySelectorAll(virtualNode, '*');
+
+const hasVisibleChildren = nodeAndDescendents.filter(
+	({ actualNode }) => !isHidden(actualNode)
```

Comment:
This isn't going to work. `isHidden` returns false for every empty div. If you want to check if something has content, use something like `dom.hasContentVirtual`, although we may need something like that doesn't look at the accessible name.

Reply:
```js
		(ignoreAria || !!aria.labelVirtual(elm)) ||
```